test/test_helpers/cuprite_setup.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# First, load Cuprite Capybara integration
+require "capybara/cuprite"
+
+# Then, we need to register our driver to be able to use it later
+# with #driven_by method.
+Capybara.register_driver(:cuprite) do |app|
+  Capybara::Cuprite::Driver.new(
+    app,
+    **{
+      # Enable debugging capabilities
+      inspector: true,
+      # Allow running Chrome in a headful mode by setting HEADLESS env
+      # var to a falsey value
+      headless: !ENV["HEADLESS"].in?(%w[n 0 no false])
```

Comment:
@koddsson mind following up this PR with some documentation on how to use this flag?